src/cli/commands/init.js

```diff
@@ -114,14 +121,18 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
       def = val;
     }
 
+    if (manifestKey === 'private' && privateFlag) {
+      def = true;
+    }
+
     if (def) {
-      question += ` (${def})`;
+      question += ` (${def.toString()})`;
```

Comment:
because after i add this flag, `def` could be `string` or possibly `boolean`. I want to make sure it will be a `string` when added to `question`.